apps/project/schema.py

```diff
@@ -121,6 +126,38 @@ def get_top_entity_contributor(project, Entity):
     ]
 
 
+def get_project_stats_summary(self):
+    projects = Project.get_for_member(self.context.request.user)
```

Comment:
Can you check if this change does any improvement?
```suggestion
    projects = Project.get_for_member(self.context.request.user).only('id)
```